csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1779,7 +1783,7 @@ Restart required by remediation (`restart_required`) of value type `object` with
         }
       },
 
-Additional restart information (`details`) of value type `string` with 1 or more characters provides additional information for the restart. This can include details on procedures, scope or impact.
+Optional restart information (`details`) of value type `string` with 1 or more characters provides additional information for the restart. This can include details on procedures, scope or impact.
```

Comment:
The JSON schema states the title as "Additional restart information". This is unrelated to the fact that `details` is an optional property (see line 1712).